lib/rules/sotd/pp.js

```diff
@@ -64,17 +70,25 @@ exports.check = function (sr, done) {
         var $a = sr.$(this)
         ,   href = $a.attr("href")
         ,   text = sr.norm($a.text())
+        ,   ids
         ;
         if (href === "http://www.w3.org/Consortium/Patent-Policy-20040205/" &&
             text === "5 February 2004 W3C Patent Policy") {
             foundFeb5 = true;
             return;
         }
-        if (/^http:\/\/www\.w3\.org\/2004\/01\/pp-impl\/\d+\/status$/.test(href) &&
-            text === "public list of any patent disclosures" &&
-            $a.attr("rel") === "disclosure") {
-            foundPublicList = true;
-            return;
+        if (/^http:\/\/www\.w3\.org\/2004\/01\/pp-impl\/\d+\/status(#.*)?$/.test(href) &&
+            text === "public list of any patent disclosures") {
+            ids = DELIVERER_ID_REGEX.exec(href);
+            console.log (href + ' - ' + DELIVERER_ID_REGEX);
+            console.dir(ids);
```

Comment:
Aren't these 2 lines remains from your development tests? If not, how useful is it in general? This is not sarcasm this time, I am genuinely wondering :)